dftimewolf/lib/processors/workspace_audit_timesketch.py

```diff
@@ -108,14 +126,14 @@ def _AddMessageString(self, timesketch_record: Dict[str, Any]) -> None:
           actor = (timesketch_record.get('actor_email') or
                    timesketch_record.get('actor_profileId') or
                    timesketch_record.get('actor_key'))
-          message += '"{0:s}"'.format(actor)
+          message += '{0:s}'.format(actor)
           continue
         value = timesketch_record.get(field)
         if not value:
           value = timesketch_record.get(field.lower())
         if not value:
           value = ''
-        message += '"{0:s}"'.format(value)
+        message += '{0:s}'.format(value)
```

Comment:
```suggestion
        message += value
```

Reply:
I can't guarantee that 'actor' is a string here - it could be a number (id) or something else.

Reply:
Same comment as above.

Reply:
Ack, changed